torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -26,9 +26,14 @@
 namespace torch {
 namespace jit {
 
+using debug_info_vector_type = c10::optional<std::vector<c10::IValue>>;
+
 char const* toString(OpCode op);
 
 namespace {
+
+using func_tuple_type = std::pair<IValue, c10::optional<IValue>>;
```

Comment:
I think it will be fine to not generate type alias, unless it is too long. This one is also not very informative.